manuscript/the-meat.txt

```diff
@@ -94,39 +94,39 @@ We start with a blank component called `Histogram`.
+Once more, we required external libraries on top and started our component with a simple `render` function. On the bottom, we added it to the exports. This will allow other parts of the codebase to use it as `require(‘drawers.jsx’).Histogram`.
 
-In the `render` method we created a grouping, `<g>`, element that will hold our histogram together and used the `transform` property to move it into place. The reason we did some ugly string concatenation, is that we can’t mix string and variable attribute values.
+In the `render` method, we created a grouping, `<g>`, element that will hold our histogram together. We used the `transform` property to move it into place. The reason we did some ugly string concatenation is that we can’t mix string and variable attribute values.
 
-But we do want to get `topMargin` via properties given to our component. We access those with `this.props` and they make our components more flexible.
+However, we do want to get `topMargin` via properties given to our component. We access those with `this.props` and they make our components more flexible.
 
-We're going to build our histogram with d3.js’s built-in histogram layout. It’s smart enough to do everything on its own. We just have to giv it some data, and some configuration.
+We're going to build our histogram with d3.js’s built-in histogram layout. It’s smart enough to do everything on its own. We just have to give it some data and some configuration.
 
 We add three methods to our `Histogram` component to manage d3.js:
 
 {crop-start-line=26,crop-end-line=45,linenos=off,lang=jsx}
 <<[D3.js management functions](code_samples/drawers.jsx)
 
-This is the cleanest approach I’ve found to make d3.js and React work together. We’ll use it for all our d3.js-enabled components.
-	
+This is the cleanest approach I’ve found to make d3.js and React work together. We’ll use it for all of our d3.js-enabled components.
+
 The problem we're facing is that the way d3.js achieves its declarative nature is through objects. When you call a d3.js function, it's usually going to have some internal state.
 
-Which is great when you want to change how it behaves by daisy chaining getters and setters. But it causes problems with React because it means we have to make sure the internal state is always up to date.
+This is great when you want to change how it behaves by daisy-chaining getters and setters, but it causes problems with React because it means we have to make sure the internal state is always up-to-date.
 
-We do this by relying on `componentWillMount`, to set the defaults, and then `componentWillRecieveProps`, to update them when necessary.
+We do this by relying on `componentWillMount` to set the defaults, and then `componentWillRecieveProps` to update them when necessary.
```

Comment:
Let's change the first part to `We'll use componentWillMount to set the defaults`